src/docs/index.md

```diff
@@ -35,9 +36,10 @@ Some of the new content includes:
 * Deep dive on Flutter internals,
   [Inside Flutter](/docs/resources/inside-flutter)
 * [Technical videos](/docs/resources/videos) page
+* [State management](/docs/development/data-and-backend/state-mgmt) landing page
 * How to connect [a native debugger _and_
   a Dart debugger to your app](/docs/testing/oem-debuggers)
-  (iOS info in process)
+  (not yet complete)
```

Comment:
Do you want to instead wrap this list item in a {% comment %} given that it isn't complete?